client/__tests__/e2e/test_config.yaml

```diff
@@ -2,8 +2,8 @@ server:
   app:
     force_https: true
     port: 5005
-    debug: true
-    verbose: true
+    debug: false
```

Comment:
if debug is true then verbose will be set to true. They must both be false to disable verbose